contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -91,24 +93,19 @@ contract WithdrawalQueue {
     /// - N after upgrading contract from the previous version (after calling finalize_vN())
     bytes32 internal constant CONTRACT_VERSION_POSITION = keccak256("lido.WithdrawalQueue.contractVersion");
 
-    /// Lido DAO Agent contract address
-    /// Used to call administrative levers
-    bytes32 internal constant LIDO_DAO_AGENT_POSITION = keccak256("lido.WithdrawalQueue.lidoDAOAgent");
-
-    /// Requests placement resume/pause control storage slot
-    bytes32 internal constant REQUESTS_PLACEMENT_RESUMED_POSITION =
-        keccak256("lido.WithdrawalQueue.requestsPlacementResumed");
+    /// Withdrawal queue resume/pause control storage slot
+    bytes32 internal constant RESUMED_POSITION = keccak256("lido.WithdrawalQueue.resumed");
 
     /// Lido stETH token address to be set upon construction
     address public immutable STETH;
     /// Lido wstETH token address to be set upon construction
     address public immutable WSTETH;
 
-    /**
-     * @notice All state-modifying calls are allowed only from owner protocol.
-     * @dev should be Lido
-     */
-    address payable public immutable OWNER;
+    // ACL
+
+    bytes32 public constant PAUSE_ROLE = keccak256("PAUSE_ROLE");
+    bytes32 public constant RESUME_ROLE = keccak256("RESUME_ROLE");
+    bytes32 public constant FINALIZE_ROLE = keccak256("FINALIZE_ROLE");
```

Comment:
Would like to ponder if we need finalizer role or just set Lido address, but good to go as it is.

Reply:
Hypothetically, it can be assigned to some other intermediary contract, to oracle or even be invoked manually. So I'd like to have some agility here. At least, assuming that we have a generalised part for handling roles here.